src/libcollectionstest/string.rs

```diff
@@ -450,3 +450,30 @@ fn bench_exact_size_shrink_to_fit(b: &mut Bencher) {
         r
     });
 }
+
+#[bench]
+fn bench_from_str(b: &mut Bencher) {
+    let s = "Hello there, the quick brown fox jumped over the lazy dog! \
+             Lorem ipsum dolor sit amet, consectetur. ";
+    b.iter(|| {
+        test::black_box(String::from_str(s));
```

Comment:
It's ok to actually omit these calls to `test::black_box`, whatever is returned from the closure will be passed to `black_box` automatically (so these could all just return the string)